Hotfix - Exclude sticky post from "New Posts Notification" campaign

mehedimi opened the pull request.

tareq1988 commented:
I think in the query, you can pass `ignore_sticky_posts => true`, it should ignore the sticky posts.

mehedimi commented:
I have removed `post__not_in` and added only `ignore_sticky_posts` and it doesn't work. So read this answer https://wordpress.stackexchange.com/a/260951/224962. What's your opinion about this answer?
